### PR TITLE
Fixed #30947 -- Changed tuples to lists in model Meta options in django.contrib modules.

### PR DESCRIPTION
**Pull request by jdufresne**
The Django "Model Meta options" docs provide examples and generally point the reader to use lists for the `unique_together` and `ordering` options. Follow our own advice for contrib models.

More generally, lists should be used for homogeneous sequences of arbitrary lengths of which both `unique_together` and `ordering` are.

https://code.djangoproject.com/ticket/30947

### CODE REVIEW

**Comment by jdufresne** [comment]
The choice of list is explained quite well in #11267 and acknowledges these differences.

**Comment by jdufresne** [comment]
Added ticket https://code.djangoproject.com/ticket/30947

**Comment by codingjoe** [comment]
@jdufresne one last thing, even tho this is a small change, it's best to add at least a one liner to the release notes. `3.1.txt` would be the right one.

**Comment by felixxm** [comment]
@jdufresne Thanks :+1: I added a separate commit with small docs changes and added release notes.

Migrations are required for changes in `Meta.ordering`.

**Comment by jdufresne** [comment]
Updated the migrations.

As there is no actual schema change (or behavioral change really), I modified the old migrations instead of creating new ones. This avoids unnecessary processing of additional migrations.

File: django/contrib/contenttypes/migrations/0001_initial.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
                 ('model', models.CharField(max_length=100, verbose_name='python model class name')),
             ],
             options={
-                'ordering': ('name',),
+                'ordering': ['name'],
```
Comment (felixxm) [inline django/contrib/contenttypes/migrations/0001_initial.py]: This change is unrelated, I'm going to revert it.

Comment (jdufresne) [reply django/contrib/contenttypes/migrations/0001_initial.py]: No problem, but do you mind explaining a bit? Above you requested changes to the migrations as well.
